numpy/linalg/linalg.py

```diff
@@ -99,7 +96,7 @@ def _raise_linalgerror_svd_nonconvergence(err, flag):
     raise LinAlgError("SVD did not converge")
 
 def get_linalg_error_extobj(callback):
-    extobj = list(_linalg_error_extobj)
```

Comment:
This line is needed to make a copy, although granted that was unclear

Reply:
Done. added comment too.